### ISSUE TITLE
Feather: Show descriptions for variables and enums with JSdoc

### ISSUE THREAD

**Issue by thennothinghappened** [YoYoGames/GameMaker-Bugs#5069]
### Is your feature request related to a problem?

It's nice that feather currently shows descriptions in tooltips for functions and methods, but it doesn't do the same for enums and variables and such.

Consider the following snippet:
```js
/// State type for objects or something.
enum State {
    /// Default blank state.
    None,
    /// Some other state.
    Idk
}

/// Current state of this instance.
/// Multi-line!
state = State.None;

/// Some instance method!
handler = function() {
	
}
```

Only one of these three gets a tooltip!

**Method (gets description in tooltip):**
<img width="297" alt="image" src="https://github.com/YoYoGames/GameMaker-Bugs/assets/35243139/bf59de7b-ff5f-427b-9f98-84269443fa27">

**Enum (no description in tooltip):**
<img width="426" alt="image" src="https://github.com/YoYoGames/GameMaker-Bugs/assets/35243139/bcec875e-3e71-4462-934d-fb6e88332928">

**Instance Variable (no description in tooltip):**
<img width="375" alt="image" src="https://github.com/YoYoGames/GameMaker-Bugs/assets/35243139/f43b0751-ad8c-4564-8e36-0c94790efee0">

### Describe the solution you'd like

It'd be cool if feather showed descriptions for these in tooltips, since its something I write a lot myself regardless (and then have to check back manually at the declaration to see) and is very valuable to libraries and such.

**Comment by gnysek** [comment]
It could also show their values.

**Comment by thennothinghappened** [comment]
> It could also show their values.

Agreed but I think that's probably a separate request than showing tooltip descriptions

**Comment by yerumaku** [comment]
> It could also show their values.

If the enumeration has flags, then you need to think about the display format, because just large numbers are not informative.

**Comment by gnysek** [comment]
Still, displaying it in tooltip could look better than in case of same large number in dropdown (where I would wish to also see those values, but that would be separate FR).

**Comment by bocoboc** [comment]
👍Enum tooltip was already working in the first versions of GM Studio 2, then it broke and I issued a ticket over 2 years ago and it got fixed for like two versions 😄 . Did anyone erase that feature by accident? It can be annoying when you just want to quickly see all enum value names without proceeding to its definition.

Please make it work again, at least for Code Editor 2.
